Core/Node-API/Include/Shared/napi/napi-inl.h

```diff
@@ -19,6 +19,12 @@
 #include <type_traits>
 #include <utility>
 
+#if defined(__clang__) || defined(__GNUC__)
+#define NAPI_NO_SANITIZE_VPTR __attribute__((no_sanitize("vptr")))
+#else
+#define NAPI_NO_SANITIZE_VPTR
+#endif
+
```

Comment:
Where did these changes come from? We try to ensure these headers match the original unless it is documented as a change. Search for `// [BABYLON-NATIVE-ADDITION]`